theia-turn/workspace/tucm_example.turn

```diff
@@ -24,7 +24,7 @@ map OrExplicit {
 
 map And {
     start s -> X r1 -> and {
-        * -> X r2 -> ;
```

Comment:
General note, I'm thinking of adding the `tucm_example.turn` file to our gitignore as I don't think we should modify it's contents (i.e. it should remain consistent across branches). `sample.turn` can be our file to messaround with on a branch-by-branch basis.
Thoughts?